example_notebooks/Process_fake_image_data.ipynb

```diff
@@ -0,0 +1,346 @@
+{
```

Comment:
I am not sure exactly what this file does, or why it is necesary. The fact that it has filepaths in it means it should not be included in a package like scikit-beam/rixs. @mpmdean can you describe what this is for and why it is necesary?

Reply:
Ok, I understand what it is for now and how it could be useful. 

We will need to test this, because I see some file paths in it so I think it may not run for anyone but you.

Reply:
I will look at the notebook again, but there are no file paths in it.